connectors/v2/driveplayer.com.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+/* global Connector */
+Connector.playerSelector = '.jp-play';
```

Comment:
`Connector.playerSelector` should be defined as `.jp-audio`. Now it's just a play button selector.